src/main.rs

```diff
@@ -1,21 +1,63 @@
+//! Default Compute@Edge template program.
+
+use fastly::http::response::Builder as ResponseBuilder;
 use fastly::http::{HeaderValue, Method, StatusCode};
 use fastly::request::CacheOverride;
 use fastly::{downstream_request, Body, Error, Request, RequestExt, Response, ResponseExt};
 use std::convert::TryFrom;
 
-const VALID_METHODS: [Method; 3] = [Method::HEAD, Method::GET, Method::POST];
+/// The name of a backend server associated with this service.
+///
+/// This should be changed to match the name of your own backend. See the the `Hosts` section of
+/// the Fastly WASM service UI for more information.
+const BACKEND_NAME: &str = "backend_name";
+
+/// The name of a second backend associated with this service.
+const OTHER_BACKEND_NAME: &str = "other_backend_name";
+
+/// The entrypoint for your application.
+///
+/// This function is triggered when your service receives a client request. This function can call
+/// [`fastly::downstream_request`][dsreq] to get the client request, and should ultimately call
+/// [`fastly::send_downstream`][send] on a [`fastly::Response`][resp] to deliver an HTTP response
+/// to the client.
+///
+/// If `main` returns an error, a [500 Internal Server Error][err] will be delivered to the client.
+///
+/// [dsreq]: ../fastly/request/fn.downstream_request.html
+/// [err]: https://tools.ietf.org/html/rfc7231#section-6.6.1
+/// [resp]: ../fastly/struct.Response.html
+/// [send]: ../fastly/response/trait.ResponseExt.html#method.send_downstream
+fn main() -> Result<(), Error> {
+    let req = downstream_request()?;
+    let resp = match handle_request(req) {
```

Comment:
A comment on this line might be nice - pointing the user to the function that follows.

Reply:
Good eye! This ends up being outdated via `8176c42`. Let me know what you think!